appveyor.yml

```diff
@@ -54,15 +60,20 @@ install:
   - nuget install simbody -Version 0.0.0 -ExcludeVersion -OutputDirectory C:\
 
 build_script:
+  ## Superbuild dependencies.cd 
+  - mkdir %OPENSIM_DEPENDENCIES_BUILD_DIR%
+  - cd %OPENSIM_DEPENDENCIES_BUILD_DIR%
+  - cmake %OPENSIM_DEPENDENCIES_SOURCE_DIR% -G"Visual Studio 14 2015 Win64" -DCMAKE_INSTALL_PREFIX=%OPENSIM_DEPENDENCIES_INSTALL_DIR%
```

Comment:
`-DSUPERBUILD_SIMBODY=OFF` could go here too; simbody binaries are set up for appveyor as well (the `nuget install simbody` line). I'm fine with migrating to a new binary artifact mechanism (which I think you are planning), but in the interim, can we use the binary system that exists now?